ui/src/app/analytics/analytics.component.spec.ts

```diff
@@ -19,7 +19,7 @@ xdescribe('AnalyticsComponent', () => {
     fixture.detectChanges();
   });
 
-  it('should be created', () => {
+  xit('should be created', () => {
```

Comment:
What's a reason to disable test?

Reply:
@ghillert update your `beforeEach` with this code segment and the test will work:
```
  beforeEach(async(() => {
    activeRoute = new MockActivatedRoute();
    TestBed.configureTestingModule({
      declarations: [ AnalyticsComponent ],
      imports: [
      RouterTestingModule.withRoutes([])
      ],
      providers: [
        { provide: ActivatedRoute, useValue: activeRoute }
      ]
    })
    .compileComponents();
  }));
```

Reply:
@ghillert   update your `beforeEach` with the segment below and the test will work.
```
beforeEach(async(() => {
    activeRoute = new MockActivatedRoute();
    TestBed.configureTestingModule({
      declarations: [ AnalyticsComponent ],
      imports: [
      RouterTestingModule.withRoutes([])
      ],
      providers: [
        { provide: ActivatedRoute, useValue: activeRoute }
      ]
    })
    .compileComponents();
  }));
```